assets/model_monitoring/components/src/model_data_collector_preprocessor/store_url.py

```diff
@@ -57,11 +57,15 @@ def _get_url(self, scheme=None, store_type=None, relative_path=None) -> str:
 
     def get_credential(self) -> Union[str, ClientSecretCredential, AzureSasCredential, None]:
         """Get credential for this store url."""
-        def get_default_credential():
-            return DefaultAzureCredential() if self._is_secure() else None
-
         if not self._datastore:
-            return get_default_credential()
+            if self._is_secure():
+                raise InvalidInputError(
+                    "abfss, wasbs and https URLs are credential less and are NOT supported by Model Monitoring job, "
+                    "please use azureml file system path with credential data store, e.g. "
+                    "azureml://datastores/<credential_datastore_name>/paths/<path_to_data>")
+            else:
```

Comment:
What's this case?

Reply:
as the err msg indicates, it's the abfss, wasbs and https URLs, these URLs are essentially credential-less URL and we don't support for now.

Reply:
raise error for all abfs(s), wasb(s) and http(s) URL